lib/Dialect/LLHD/Transforms/TemporalRegions.h

```diff
@@ -20,33 +20,37 @@
 namespace circt {
 namespace llhd {
 
-struct TemporalRegionAnalysis {
-  using BlockMapT = DenseMap<Block *, int>;
-  using TRMapT = DenseMap<int, SmallVector<Block *, 8>>;
-
+class TemporalRegionAnalysis {
```

Comment:
Good catch to change this to a class (because the struct has private fields).

Reply:
Thx